app/models/opensearch.rb

```diff
@@ -177,7 +198,7 @@ def source_array(param)
     param.each do |source|
       sources << {
         term: {
-          source: source
+          source:
```

Comment:
Same question here as above - this is now being called implicitly?

Reply:
Yup